### PR TITLE
if firefox and windows -> http

### PR DESCRIPTION
**Pull request by SebastienValla**

### CODE REVIEW

**Review by 0xMazout — APPROVED** [review]
if its working lets go

File: int/api/html/front/website.js [hunk 1]
```diff
@@ -162,7 +162,9 @@ async function tableInsert(resp, count) {
         .getElementById("website-deployers-table")
         .getElementsByTagName("tbody")[0];
     const row = tBody.insertRow(-1);
-    const url = `//${resp.name}.massa:${window.location.port}/`;
+    const url = isWindowsAndFirefox()
+        ? `http://${resp.name}.massa:${window.location.port}/`
+        : `https://${resp.name}.massa:${window.location.port}/`;
```
Comment (Thykof) [inline int/api/html/front/website.js:167]: Duplicated code here, maybe you could add a function `getUrl()`

File: int/api/html/front/registry.js [hunk 2]
```diff
@@ -24,7 +24,10 @@ function tableInsert(website) {
         .getElementById("website-deployers-table")
         .getElementsByTagName("tbody")[0];
     const row = tBody.insertRow(-1);
-    const url = `//${website[NAME]}.massa:${window.location.port}/`;
+
+    const url = isWindowsAndFirefox()
+        ? `http://${website[NAME]}.massa:${window.location.port}/`
+        : `https://${website[NAME]}.massa:${window.location.port}/`;
```
Comment (gregLibert) [inline int/api/html/front/registry.js:30]:
```suggestion
    const protocol = isWindowsAndFirefox() ? "http" : "https";
    const url = `${protocol}://${website[NAME]}.massa:${window.location.port}/`;
```

File: int/api/html/front/website.js (hunk 1 above)
Comment (gregLibert) [reply int/api/html/front/website.js:167]: same as above

**Review by gregLibert — CHANGES_REQUESTED** [review]

**Review by gregLibert — APPROVED** [review]
:+1:

**Pull request merged**